syncapi/synctypes/clientevent.go

```diff
@@ -141,6 +141,18 @@ func ToClientEvent(se gomatrixserverlib.PDU, format ClientEventFormat, userIDFor
 		Redacts:        se.Redacts(),
 	}
 
+	// RoomVersion V11 moved the "redacts" field to "content", so add it there as well
+	// For compatibility with older clients, we keep the "redacts" top-level field.
+	// TODO: At which point do we remove "redacts" from the top-level?
+	if se.Type() == spec.MRoomRedaction && se.Version() == gomatrixserverlib.RoomVersionV11 && ce.Redacts != "" {
```

Comment:
I don't think this should have the `se.Version() == gomatrixserverlib.RoomVersionV11` check.
I think the logic is inverted - ie. for any room version where there is a top level redacts field we should be copying that into the content for clients.
I think we can just remove the version check entirely.
```
For improved compatibility with newer clients, servers should add a redacts property to the content of m.room.redaction events in older room versions when serving such events over the Client-Server API.
```